src/views/notifications/CloseAllPositionsNotification.tsx

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components';
+
+import { AbacusOrderStatus } from '@/constants/abacus';
+import { ButtonAction, ButtonSize } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+import { LocalCloseAllPositionsData } from '@/constants/trade';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useSubaccount } from '@/hooks/useSubaccount';
+
+import { Button } from '@/components/Button';
+import { Details } from '@/components/Details';
+import { LoadingSpinner } from '@/components/Loading/LoadingSpinner';
+// eslint-disable-next-line import/no-cycle
+import { Notification, NotificationProps } from '@/components/Notification';
+
+import { OrderStatusIcon } from '../OrderStatusIcon';
+
+type ElementProps = {
+  localCloseAllPositions: LocalCloseAllPositionsData;
+};
+
+export const CloseAllPositionsNotification = ({
+  isToast,
+  localCloseAllPositions,
+  notification,
+}: NotificationProps & ElementProps) => {
+  const stringGetter = useStringGetter();
+  const { closeAllPositions } = useSubaccount();
```

Comment:
tiny nit suggestion to add `{submittedOrderClientIds, filledOrderClientIds, failedOrderClientIds} = localCloseAllPositions;`